website/docs/r/nsxt_edgegateway_l2_vpn_tunnel.html.markdown

```diff
@@ -0,0 +1,140 @@
+---
+
+# vcd\_nsxt\_edgegateway\_l2\_vpn\_tunnel
+
+Supported in provider *v3.11+* and VCD *10.2+* with NSX-T
+
+Provides a resource to manage NSX-T Edge Gateway L2 VPN Tunnel sessions and their configurations.
+
+## Example Usage (Both server and client tunnel sessions connecting both Edge Gateways)
+
+```hcl
+resource "vcd_nsxt_edgegateway_l2_vpn_tunnel" "server-session" {
+  org = "datacloud"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.server-testing.id
+
+  name        = "server-session"
+  description = "example description"
+
+  session_mode             = "SERVER"
+  enabled                  = true
+  connector_initiator_mode = "ON_DEMAND"
+
+  # must be sub-allocated on the Edge Gateway
+  local_endpoint_ip  = "10.10.50.2"
+  tunnel_interface   = "192.168.0.1/24"
+  remote_endpoint_ip = "1.2.2.3"
+
+  stretched_network {
+    network_id = data.vcd_routed_network_v2.test_network_server.id
+  }
+
+  pre_shared_key = "secret_passphrase"
+}
+
+resource "vcd_nsxt_edgegateway_l2_vpn_tunnel" "client-session" {
+  org = "datacloud"
+
+  # Note that this is different, as one edge gateway can only function
+  # in SERVER or CLIENT mode.
+  edge_gateway_id = data.vcd_nsxt_edgegateway.client-testing.id
+
+  name        = "client-session"
+  description = "example description"
+
+  session_mode = "CLIENT"
+  enabled      = true
+
+  # must be sub-allocated on the Edge Gateway
+  local_endpoint_ip  = "101.22.30.3"
+  remote_endpoint_ip = "1.2.2.3"
+
+  stretched_network {
+    network_id = data.vcd_routed_network_v2.test_network_client.id
+    # CLIENT sessions need to define a tunnel ID for every stretched network
+    tunnel_id = 1
+  }
+
+  stretched_network {
+    network_id = data.vcd_routed_network_v2.test_network_client_other.id
+    tunnel_id  = 2
+  }
+
+  # Be aware, that if there are changes in the `server-session`, the peer_code
+  # will be updated aswell, so `terraform apply` needs to be run twice
+  peer_code = vcd_nsxt_edgegateway_l2_vpn_tunnel.server-session.peer_code
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at 
+  provider level. Useful when connected as sysadmin working across different organisations
+* `edge_gateway_id` - (Required) The ID of the edge gateway (NSX-T only). 
+  Can be looked up using `vcd_nsxt_edgegateway` datasource
+* `name` - (Required) The name of the tunnel.
+* `description` - (Optional) The description of the tunnel.
+* `session_mode` - (Required) Mode of the tunnel session (SERVER or CLIENT)
+* `enabled` - (Optional) State of the `SERVER` session, always set to `true` for `CLIENT` 
+  sessions.
+* `connector_initiator_mode` - (Required for `SERVER` sessions) Mode in which 
+  the connection is formed. Only relevant to `SERVER` sessions. One of:
+	* `INITIATOR` - Local endpoint initiates tunnel setup and will also respond to 
```

Comment:
Not sure if this is an issue with quotes or a problem with GitHub, the sentence is highlighted:

<img width="701" alt="Screenshot 2023-09-28 at 17 06 05" src="https://github.com/vmware/terraform-provider-vcd/assets/11330689/f907fed5-d8d8-45db-95a3-0cc58cfbc6f6">

Reply:
<img width="676" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/16798732/07a9a323-6514-41f9-b335-0918e29fada4">
It renders fine on Terraform's [doc-preview](https://registry.terraform.io/tools/doc-preview) page, so it's probably just GitHub